src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c

```diff
@@ -3421,6 +3448,8 @@ static int mb_scheduler_remove(struct platform_device *pdev)
 	}
 	mutex_destroy(&exec->exec_lock);
 
+	abort_all_clients(pdev);
```

Comment:
We aborted all clients, without releasing them. But we freed exec and did platform_set_drvdata(pdev, NULL); So, later when destroy_client() is called, can we still free the client data structure?